tardis/atomic/base.py

```diff
@@ -9,9 +9,9 @@ class AtomicData(object):
     def from_hdf(cls, fname):
         return cls(pd.HDFStore(fname, mode='r'))
 
-    def __init__(self, hdf_store, data_types):
+    def __init__(self, hdf_store, data_types=self.all_data_types):
```

Comment:
this won't quite work. you can't access `self` in the initializer. the `data_types` should also be given by the user. so for now, I suggest leaving it as is.